ui/app/templates/vault/cluster/init.hbs

```diff
@@ -40,27 +52,21 @@
       </div>
       <div class="box is-marginless is-shadowless">
         <div class="field is-grouped-split">
-          {{#if model.sealed}}
-            <div class="control">
+          {{#if (and model.sealed (not keyData.recovery_keys))}}
+            <div data-test-advance-button class="control">
               {{#link-to 'vault.cluster.unseal' model.name class="button is-primary"}}
-              Continue to Unseal
+                Continue to Unseal
               {{/link-to}}
             </div>
           {{else}}
-            <div class="control">
-              {{#link-to 'vault.cluster.auth' model.name class="button is-primary"}}
-              Continue to Authenticate
+            <div data-test-advance-button class="control">
+              {{#link-to 'vault.cluster.auth' model.name class=(concat (if model.sealed 'is-loading ' '') 'button is-primary') disabled=model.sealed}}
+                Continue to Authenticate
               {{/link-to}}
             </div>
           {{/if}}
-          <DownloadButton
-            @data={{keyData}}
-            @filename={{keyFilename}}
-            @mime="application/json"
-            @extension="json"
-            @class="button is-ghost"
-            @stringify={{true}}
-            >
+          <DownloadButton @data={{keyData}} @filename={{keyFilename}} @mime="application/json" @extension="json" @class="button is-ghost"
+            @stringify={{true}}>
```

Comment:
This is some weird formatting.

Reply:
Ugh yes js-beautify's default which VSCode uses does "break attrs only after max line length" by default - fixed it to break all attrs which is also kinda weird, but more easily read imo.